prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -111,7 +111,16 @@ public async Task<EnrolleeViewModel> GetEnrolleeAsync(int enrolleeId)
                 .DecompileAsync()
                 .SingleOrDefaultAsync();
 
-            return _mapper.Map<EnrolleeViewModel>(dto);
+            var linkedGpid = await _context.EnrolleeLinkedEnrolments
+                .Where(ele => ele.EnrolleeId == enrolleeId)
+                .Select(ele => ele.UserProvidedGpid)
+                .SingleOrDefaultAsync();
```

Comment:
_theoretically_ there is the possibility of multiple `EnrolleeLinkedEnrolments` with a matching EnrolleeId. The `.Where` should be updated to       
```C#
.Where(ele => ele.EnrolleeId == enrolleeId
    && ele.PaperEnroleeId.HasValue)
```